lib/matplotlib/axis.py

```diff
@@ -57,26 +57,12 @@ class Tick(martist.Artist):
     """
     def __init__(
         self, axes, loc, *,
-        size=None,  # points
-        width=None,
-        color=None,
-        tickdir=None,
-        pad=None,
-        labelsize=None,
-        labelcolor=None,
-        labelfontfamily=None,
-        zorder=None,
-        gridOn=None,  # defaults to axes.grid depending on axes.grid.which
-        tick1On=True,
-        tick2On=True,
-        label1On=True,
-        label2On=False,
-        major=True,
-        labelrotation=0,
-        grid_color=None,
-        grid_linestyle=None,
-        grid_linewidth=None,
-        grid_alpha=None,
+        size=None, width=None, color=None, major=True, pad=None, zorder=None,  # global
+        tickdir=None, tick1On=True, tick2On=True,  # tick prop
+        labelsize=None, labelcolor=None, labelrotation=0,  # label prop
```

Comment:
I think a keyword got lost in the shuffle?

```suggestion
        labelsize=None, labelcolor=None, labelrotation=0,  labelfontfamily=None,  # label prop
```

Reply:
oh sorry, I was trying to find where this parameter was from because of the linting error but wasn't finding it.
Thanks for that. I'm fixing right now